packages/babel-preset-sui/lib/index.js

```diff
@@ -1,12 +1,41 @@
-const isInstalled = require('./is-installed')
-const cleanList = require('./clean-list')
+console.log('new babel')
 
-module.exports = {
-  presets: cleanList([
+function plugins(api, opts) {
+  return [
+    require('@babel/plugin-syntax-dynamic-import').default,
+    [require('@babel/plugin-proposal-decorators').default, {legacy: true}],
+    [require('@babel/plugin-proposal-class-properties').default, {loose: true}],
     [
-      'env',
+      require('babel-plugin-transform-react-remove-prop-types').default,
+      {
+        wrap: true
+      }
+    ],
+    [
+      require('@babel/plugin-proposal-object-rest-spread').default,
+      {useBuiltIns: true} // asume Object.assign is available by browser or polyfill
+    ],
+    [
+      require('@babel/plugin-transform-runtime').default,
+      {
+        corejs: false,
+        regenerator: true
+      }
+    ]
+  ]
+}
+
+function presets(api, opts) {
+  return [
+    [
+      require('@babel/preset-env').default,
       {
         debug: false,
+        ignoreBrowserslistConfig: true,
+        loose: true,
+        modules: false,
+        // Exclude transforms that make all code slower
+        exclude: ['transform-typeof-symbol'],
         targets: {
           node: '6.0.0',
```

Comment:
Maybe we can set up this at Node 8

Reply:
I wanted to keep compatibility for now, to avoid doing this step forward in the same PR. Also, FC is still using Node 6 in pro.